aladdin.sh

```diff
@@ -30,6 +31,15 @@ DEFAULT_MINIKUBE_MEMORY=4096
 
 source "$SCRIPT_DIR/shared.sh" # to load _extract_cluster_config_value
 
+# If $MANAGE_MINIKUBE is true, then all commands will be executed
+# using the external 'minikube' command.
+# https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-command
+# Otherwise calls to minikube will be no-op
+function minikube() {
+    if "$MANAGE_MINIKUBE"; then
```

Comment:
let's use this conditional form throughout, rather than `if [ "$MANAGE_MINIKUBE" = false ] ; then`

Reply:
Pushed the commit with this suggestion implemented.